### PR TITLE
[DOCS] torch.compile examples

### PR DESCRIPTION
**Pull request by kblaszczak-intel**

### CODE REVIEW

**Review by tsavina — APPROVED** [review]

File: docs/articles_en/openvino-workflow/torch-compile.rst [hunk 1]
```diff
@@ -20,6 +20,179 @@ By default, Torch code runs in eager-mode, but with the use of ``torch.compile``
...
+
+         prompt = "a photo of an astronaut riding a horse on mars"
+         image = pipe(prompt).images[0]
+
+         image.save("astronaut_rides_horse.png")
+
+
+   .. tab-item:: Stable-Diffusion-3
+
+      .. code-block:: sh
```
Comment (rkazants) [inline docs/articles_en/openvino-workflow/torch-compile.rst:52]:
```suggestion
      .. code-block:: py
```

File: docs/articles_en/openvino-workflow/torch-compile.rst [hunk 2]
```diff
@@ -20,6 +20,179 @@ By default, Torch code runs in eager-mode, but with the use of ``torch.compile``
...
+             negative_prompt="",
+             num_inference_steps=28,
+             guidance_scale=7.0,
+         ).images[0]
+
+         image.save('out.png')
+
+   .. tab-item:: Stable-Diffusion-XL
+
+      .. code-block:: sh
```
Comment (rkazants) [inline docs/articles_en/openvino-workflow/torch-compile.rst:72]:
```suggestion
      .. code-block:: py
```

File: docs/articles_en/openvino-workflow/torch-compile.rst [hunk 3]
```diff
@@ -20,6 +20,179 @@ By default, Torch code runs in eager-mode, but with the use of ``torch.compile``
...
+         + pipe.vae.decode = torch.compile(pipe.vae.decode, backend="openvino") #Optional
+
+         prompt = "a close-up picture of an old man standing in the rain"
+         image = pipe(prompt, num_inference_steps=5, guidance_scale=8.0).images[0]
+         image.save("result.png")
+
+
+   .. tab-item:: Llama-3.2-1B
+
+      .. code-block:: sh
```
Comment (rkazants) [inline docs/articles_en/openvino-workflow/torch-compile.rst:92]:
```suggestion
      .. code-block:: py
```

File: docs/articles_en/openvino-workflow/torch-compile.rst [hunk 4]
```diff
@@ -20,6 +20,179 @@ By default, Torch code runs in eager-mode, but with the use of ``torch.compile``
...
+             tokenizer=tokenizer,
+             max_new_tokens=64
+         )
+         result = pipe(prompt)
+         print(result[0]['generated_text'])
+
+
+   .. tab-item:: Llama-2-7B-GPTQ
+
+      .. code-block:: sh
```
Comment (rkazants) [inline docs/articles_en/openvino-workflow/torch-compile.rst:122]:
```suggestion
      .. code-block:: py
```

File: docs/articles_en/openvino-workflow/torch-compile.rst [hunk 5]
```diff
@@ -20,6 +20,179 @@ By default, Torch code runs in eager-mode, but with the use of ``torch.compile``
...
+             tokenizer=tokenizer,
+             max_new_tokens=64
+         )
+         result = pipe(prompt)
+         print(result[0]['generated_text'])
+
+
+   .. tab-item:: Chatglm-4-GPTQ
+
+      .. code-block:: sh
```
Comment (rkazants) [inline docs/articles_en/openvino-workflow/torch-compile.rst:152]:
```suggestion
      .. code-block:: py
```

File: docs/articles_en/openvino-workflow/torch-compile.rst [hunk 6]
```diff
@@ -20,6 +20,186 @@ By default, Torch code runs in eager-mode, but with the use of ``torch.compile``
...
+
+               + pipe.text_encoder = torch.compile(pipe.text_encoder, backend="openvino") #Optional
+               + pipe.unet = torch.compile(pipe.unet, backend="openvino")
+               + pipe.vae.decode = torch.compile(pipe.vae.decode, backend="openvino") #Optional
+
+               prompt = "a close-up picture of an old man standing in the rain"
+               image = pipe(prompt, num_inference_steps=5, guidance_scale=8.0).images[0]
+               image.save("result.png")
+
+   .. tab-item:: Image Generation
```
Comment (ynimmaga) [inline docs/articles_en/openvino-workflow/torch-compile.rst:93]: These are text generation models. can you please change the tab to Text generation?

**Pull request merged**